sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -1353,6 +1353,15 @@ class HiveDDLSuite
       sql("INSERT INTO t SELECT 2, 'b'")
       checkAnswer(spark.table("t"), Row(9, "x") :: Row(2, "b") :: Nil)
 
+      Seq(10 -> "y").toDF("i", "j")
```

Comment:
please add a new test, to append to a hive table, also test append to a data source table with hive provider and check the error message